### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by joliveros**

### CODE REVIEW

File: public/dialer.js [hunk 1]
```diff
@@ -41,16 +41,17 @@ App.IndexController = Ember.Controller.extend({
...
-      { name: 'Great Britain', cc: '44', code: 'gb' },
+      { name: 'Brazil', cc: '55', code: 'br' },
       { name: 'Colombia', cc: '57', code: 'co' },
       { name: 'Ecuador', cc: '593', code: 'ec' },
       { name: 'Estonia', cc: '372', code: 'ee' },
       { name: 'Germany', cc: '49', code: 'de' },
+      { name: 'Great Britain', cc: '44', code: 'gb' },
       { name: 'Hong Kong', cc: '852', code: 'hk' },
       { name: 'Ireland', cc: '353', code: 'ie' },
+      { name: 'Mexico', cc: '52', code: 'mx' },
```
Comment (acamino) [inline public/dialer.js:53]: 😉

File: public/index.html [hunk 2]
```diff
@@ -109,13 +109,12 @@
 
     <!-- Status logging -->
     <div class="log">{{ logtext }}</div>
-    <p>{{ identity }}</p>
 
   </script>
 
-  <script src="//media.twiliocdn.com/sdk/js/client/v1.3/twilio.min.js"></script>
+  <script src="//media.twiliocdn.com/sdk/js/client/v1.4/twilio.min.js"></script>
```
Comment (acamino) [inline public/index.html]: I recall we agreed on using the Twilio.js client version 1.3. v1.4 is still in beta.
https://www.twilio.com/docs/api/client/changelog-14-beta

**Review by acamino — COMMENTED** [review]
Thanks @joliveros! Almost done!

**Comment by joliveros** [comment]
@acamino Looks like this is good for another pass. Let me know if we need any other changes.

thank you!

**Comment by acamino** [comment]
Thanks @joliveros!

**Pull request merged**